packages/components-js/tests/unit/fixtures/stats.json

```diff
@@ -1,272 +1,272 @@
 [
   {
     "chunkShortName": "accordion",
-    "size": 4543,
-    "gzipSize": 1980
+    "size": 4919,
```

Comment:
Still a mystery why several components grew quite significantly up too 600 bytes.

Reply:
We checked the generated chunks and stencil added keys to the slots which correlated to the increase in size